lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx

```diff
@@ -4,116 +4,115 @@
-
-    expect(none).toHaveProp('title', 'None');
-    expect(low).toHaveProp('title', 'Low');
-    expect(moderate).toHaveProp('title', 'Moderate');
-    expect(severe).toHaveProp('title', 'Severe');
-    expect(critical).toHaveProp('title', 'Critical');
+  describe('should have default tooltips', function() {
+    describe('when policyThreatLevel prop is provided', function() {
+      allThreatLevelNumbers.forEach(threatNumber => {
+        it(`for policy threat level ${threatNumber}`, async function() {
+          const el = quickRender({ policyThreatLevel: threatNumber })!,
+              user = userEvent.setup(),
+              category = categoryByPolicyThreatLevel[threatNumber],
+              categoryTitleCase = category[0].toUpperCase() + category.slice(1),
+              threatIndicator = el.getByRole('img', { hidden: true, queryFallbacks: true });
+
+          await user.hover(threatIndicator);
+          const tooltip = await screen.findByRole('tooltip');
+          expect(tooltip).toHaveTextContent(categoryTitleCase);
+        });
+      });
+    });
+
+    describe('when threatLevelCategory prop is provided', function() {
+      allThreatLevelCategories.forEach(category => {
+        it(`for threat level category ${category}`, async function() {
+          const el = quickRender({ threatLevelCategory: category })!,
+              user = userEvent.setup(),
+              threatIndicator = el.getByRole('img', { hidden: true, queryFallbacks: true }),
+              categoryTitleCase = category[0].toUpperCase() + category.slice(1);
+
+          await user.hover(threatIndicator);
+          const tooltip = await screen.findByRole('tooltip');
+          expect(tooltip).toHaveTextContent(categoryTitleCase);
+        });
+      });
+    });
   });
 
-  it('should show custom tooltip title', function() {
-    const tooltip = getShallowComponent({ title: 'Extinction Level Threat' }).find(NxToolip);
+  it('should show custom tooltip title', async function() {
+    const el = quickRender({ title: 'Extinction Level Threat' }),
+        user = userEvent.setup(),
+        threatIndicator = el.getByRole('img', { hidden: true, queryFallbacks: true })!;
 
-    expect(tooltip).toHaveProp('title', 'Extinction Level Threat');
+    await user.hover(threatIndicator);
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toHaveTextContent('Extinction Level Threat');
   });
 
-  it('adds specified class names to the element', function() {
-    const icon = getShallowComponent({ className: 'foo', threatLevelCategory: 'low' }).find(NxFontAwesomeIcon);
+  describe('when presentational prop is set to true', function() {
+    let el: RenderResult;
 
-    expect(icon).toHaveClassName('foo');
-    expect(icon).toHaveClassName('nx-threat-indicator');
-    expect(icon).toHaveClassName('nx-threat-indicator--low');
-  });
+    beforeEach(() => {
+      el = quickRender({ presentational: true });
```

Comment:
Using beforeEach in this way is an anti-pattern to some people.  As an alternative, consider shadowing `quickRender` within this `describe` block:
```
const quickRender = rtlRender(NxThreatIndicator, { ...minimalProps, presentational: true });
```